docs/ios/README.md

```diff
@@ -6,38 +6,29 @@ PSPDFKit comes with open source plugins for Cordova on both [iOS](https://pspdfk
 
 ## Requirements
 
-- Xcode 11.3.1 or later
-- PSPDFKit 9.2.0 for iOS or later
+- Xcode 11.5 or later
+- PSPDFKit 9.4.0 for iOS or later
 - Cordova >= 9.0.0
-- CocoaPods >= 1.8.1
+- CocoaPods >= 1.9.3
 
 ## Installation
 
 We assume that you have [an existing Cordova project](https://cordova.apache.org/#getstarted).
 
 1. Run `cordova plugin add https://github.com/PSPDFKit/PSPDFKit-Cordova.git` to install the `pspdfkit-cordova` plugin.
-2. Update your `Podfile`: `cd platforms/ios` and open your `Podfile` in a text editor. Make sure the platform is set to iOS 11 or later, and replace `YOUR_COCOAPODS_KEY_GOES_HERE` with your own CocoaPods Key. Your `Podfile` should look like this:
+2. Update your `Podfile`: `cd platforms/ios` and open your `Podfile` in a text editor. Make sure the platform is set to iOS 12 or later. Your `Podfile` should look like this:
 
 ```diff
 source 'https://github.com/CocoaPods/Specs.git'
 - platform :ios, '9.0'
-+ platform :ios, '11.0'
-
-use_frameworks!
```

Comment:
I think keeping the lines below in the README doesn't hurt, even though you don't need to modify them anymore.

```
use_frameworks!
target 'CordovaDemo' do
	project 'CordovaDemo.xcodeproj'
	pod 'PSPDFKit', podspec: 'https://customers.pspdfkit.com/pspdfkit-ios/latest.podspec'
end
```